Makefile

```diff
@@ -82,7 +82,7 @@ audit:
 	cargo audit --ignore RUSTSEC-2020-0071
 
 spellcheck:
-	cargo spellcheck --code 1
+	cargo spellcheck --code 1 || echo "spellcheck failed. See .config/spellcheck.md for tips"
```

Comment:
This will silently ignore any spellcheck errors, also in the CI. We want the CI to fail on this.

Reply:
whoops, sorry that wasn't my intention :D